README.md

```diff
@@ -34,14 +34,26 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+    Semantics is cleanliness. Semantics is Organization. Semantics is preofessionalism. The semantics of the code is how it is percieved by other humans. It is how your code looks, feels and is deigned based on accountability of a set of standards. Semantics reinforce meaning of the information in webpages and web applications rather than merely define its presentation or look.    
+
```

Comment:
great answer here, the semantic tags also help screen reader software to correctly section and categorize the page for the visually impaired user